easybuild/framework/easyconfig/default.py

```diff
@@ -116,7 +116,7 @@
     'stop': [None, 'Keyword to halt the build process after a certain step.', BUILD],
     'tests': [[], ("List of test-scripts to run after install. A test script should return a "
                    "non-zero exit status to fail"), BUILD],
-    'unpack_options': [None, "Extra options for unpacking source", BUILD],
+    'unpack_options': ['', "Extra options for unpacking source", BUILD],
```

Comment:
if we use `[]` here as a default value, we open the door for very weird/hard to debug bugs where somewhere deep in the code we're updating this list; same argument as why using `[]` as a default value for a function argument is a bad idea, it can cause global side-effects

as to why not `None`:

```
Can't update configuration value for unpack_options, because it's not a string or list.")
```
(the `update` method in `EasyConfig` only supports updating list or string-typed values, because it appends (and doesn't want to guess the type a value should be if the previous value was `None`)